File: openapi/code_samples/PHP/menu/get.php
<?php

// GET /menu is public — no authentication required.
$ch = curl_init('https://api.cafe.redocly.com/menu');
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);

$body = curl_exec($ch);

if (curl_errno($ch)) {
    throw new RuntimeException(curl_error($ch));
}

curl_close($ch);

echo $body;

File: openapi/code_samples/C_sharp/menu/get.cs
using System;
using System.Net.Http;
using System.Threading.Tasks;

class Program
{
    static async Task Main()
    {
        using var client = new HttpClient();

        // GET /menu is public — no authentication required.
        using var response = await client.GetAsync("https://api.cafe.redocly.com/menu");
        response.EnsureSuccessStatusCode();

        var body = await response.Content.ReadAsStringAsync();
        Console.WriteLine(body);
    }
}
